samples/dotnet/Microsoft.Azure.Devices.Edge.Samples.sln

```diff
@@ -1,36 +0,0 @@
-﻿
```

Comment:
Synced with philip and we decided to remove because:
- it was causing problems with the build
- it is the only existing pattern of mini solutions